docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: '3.6'
 services:
   postgres:
-    image: postgres
+    image: postgres:12
```

Comment:
I set the version to 12 as I struggled with a Postgres syntax error in the GH action (postgres v12) that was not happening when testing locally (latest Postgres version).
It turned out I was using `CREATE OR REPLACE TRIGGER`, which was introduced in Postgres v14